spec/controllers/finders_controller_spec.rb

```diff
@@ -96,6 +96,12 @@
         get :show, params: { slug: "lunch-finder" }
         expect(response.status).to eq(406)
       end
+
```

Comment:
Can we update this test case, because the fix works, but I don't think this test case covers it.